Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "LogExpFunctions"
 uuid = "2ab3a3ac-af41-5b50-aa03-7779005ae688"
 authors = ["StatsFun.jl contributors, Tamas K. Papp <tkpapp@gmail.com>"]
-version = "0.2.4"
+version = "0.2.5"
```

Comment:
Technically I guess this PR should be considered as breaking, right? Though in practice I agree it would be annoying to tag a breaking release so maybe that's OK. (If we had reached 1.0 we could tag a minor release...)

Reply:
I think this is just a bugfix (the current behaviour is incorrect) and hence non-breaking.

Reply:
I agree, this is a bugfix.